changelogs/current.yaml

```diff
@@ -5,6 +5,10 @@ behavior_changes:
 
 minor_behavior_changes:
 # *Changes that may cause incompatibilities for some users, but should not for most*
+- area: http3/quic
```

Comment:
This isn't a user-visible behavior change. No release note neccessary.

Reply:
Are people with custom extensions not considered users for the purposes of relnotes? This change will break the build of any external QuicConnectionIdGenerator extensions. (e.g. we have one!)

Reply:
Nope, definitely not. It would take a really large effort to document every change that could break the build of any extension. Let the compiler and test coverage tell you what's wrong.

Reply:
relnote removed.